blog/2023-06-15-tauri-board-elections-and-governance-updates.mdx

```diff
@@ -0,0 +1,61 @@
+
+If you have any questions or would like more information please reach out by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Daniel Thompson-Yvetot on Discord (`@Denjell`) or email ([denjell@tauri.app](mailto:denjell@tauri.app)).
+
+---
+
+## Governance Update
+
+Tauri has always had a Working Group that's responsible for the day-to-day direction and execution, but as we've grew it became clear that we needed to evolve the Working Group. We also wanted to give new Working Group members a clear places to jump in and get started. We thought of how we could achieve this while building and iterating on the existing framework.  **With these goals in mind, we created two new concepts within the Working Group: Domains and Domain Leads.**
+
+### Domains
+
+A Domain in the Working Group represents a specific area of interest. We're starting with these 4 domains:
+
+- **Development**: Developing and maintaining the Tauri software
+- **Community**: Looks after the broader community resources and the public presence of Tauri
+- **Governance & Guidance**: Involved with all things organizational in nature
+- **Operations**: Responsible for all the tooling and infrastructure Tauri needs to get work done
+
+This gives clear entry points for a new member of the Working Group to jump into right away. It also allows us to continue growing our core competency of core Tauri development while giving additional focus to all areas which support the Tauri organization as a whole. Each of these domains will be represented by Domain Members and Domain Leads, but membership is not mutually exclusive.
+
+### Domain Leads
+
+Within a domain, there is a position for 2-3 Domain Leads. These are core individuals in the Working Group that are critical to their respective Domain. They help by setting the direction and priorities for their domain while also serving as key leaders for the Working Group as a whole.
+
+Leads are elected twice a year, in the spring and in the fall. Since this governance update is being rolled out mid-term we're appointing interim leads that have been key members of the Working Group. This fall we'll hold the first round of elections to formally elect the Domain Leads. We'll announce information for that process when we're closer to election season.
+
+---
+
+We hope that these changes provide the foundation to continue the journey the Tauri Working Group has been on while also  catalyzing the continued growth and sustainability of the organization. You can learn more about these updates in the [Governance and Guidance repo on GitHub](https://github.com/tauri-apps/governance-and-guidance). Give us your feedback on the [GitHub Discussion for this post](https://github.com/tauri-apps/tauri-docs/discussions/1116).
```

Comment:
Just a heads up the Discussion link goes to a random unrelated PR atm :laughing:

Reply:
Yeah, this _should_ be the next one that get's created if I can do numbers right ;)

Reply:
Nah it isn't because the number is shared between issues and PRs too. So this redirected to #1116